lib/screens/explore_screens/regions_screens/region_event_details/region_event_details_screen.dart

```diff
@@ -0,0 +1,141 @@
+                  ),
+                ),
+                Positioned(
+                  left: 0,
+                  top: 0,
+                  right: 0,
+                  child: Container(
+                    width: width,
+                    height: 80,
+                    decoration: const BoxDecoration(
+                      gradient: LinearGradient(
+                        colors: [Colors.black54, Colors.transparent],
+                        begin: Alignment.topCenter,
+                        end: Alignment.bottomCenter,
+                      ),
+                    ),
+                  ),
+                ),
+                Positioned(left: 0, top: 0, right: 0, child: AppBar(backgroundColor: Colors.transparent)),
+              ],
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 20.0, vertical: 16.0),
+            child: Column(
+              children: [
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                  children: [
+                    Text(
+                      DateFormat.yMMMMEEEEd().format(DateTime.parse(event.createdTime.toDate().toString())),
+                      style: Theme.of(context).textTheme.subtitle2LowEmphasis,
+                    ),
+                    Container(
+                      decoration: BoxDecoration(
+                        borderRadius: BorderRadius.circular(11.0),
+                        border: Border.all(color: AppColors.green3),
+                      ),
+                      padding: const EdgeInsets.all(4.0),
+                      child: Row(
+                        children: [
+                          const Icon(Icons.access_time, size: 14, color: AppColors.green3),
+                          const SizedBox(width: 4.0),
+                          Text(
+                            '${event.formattedStartTime} - ${event.formattedEndTime}',
+                            style: Theme.of(context).textTheme.subtitle3Green,
+                          ),
+                        ],
+                      ),
+                    ),
+                  ],
+                ),
+                Row(children: [Text(event.eventName, style: Theme.of(context).textTheme.headline7)]),
+              ],
+            ),
+          ),
+          const DividerJungle(),
+          Padding(
+            padding: const EdgeInsets.symmetric(horizontal: 20.0, vertical: 16.0),
+            child: Column(
+              children: [
+                Row(children: [Text('Details', style: Theme.of(context).textTheme.headline7)]),
+                const SizedBox(height: 16.0),
+                Row(
+                  children: [
+                    const Icon(Icons.location_on_outlined),
+                    const SizedBox(width: 11.0),
+                    Flexible(
+                      child: InkWell(
+                        onTap: () => launch(
```

Comment:
send an event to the bloc. 
have the bloc create this URL 
pass a command to the view

Reply:
OK, but tbh I don't even know if this is a clickleable link Jason I don't make that clear I just assume it.

Reply:
It should be clickable, thank you!

Reply:
Bloc created.